sklearn/metrics/tests/test_pairwise.py

```diff
@@ -595,6 +624,28 @@ def test_cosine_distances():
     assert_true(np.all(D <= 2.))
 
 
+def test_haversine_distances():
+    # Check haversine distance with distances computation
+    def slow_haversine_distances(x, y):
+        diff_lat = y[0] - x[0]
+        diff_lon = y[1] - x[1]
+        a = np.sin(diff_lat / 2) ** 2 + \
```

Comment:
Please use parentheses rather than \ to continue lines

Reply:
done